desc/plotting.py

```diff
@@ -3843,3 +3844,106 @@ def plot_logo(save_path=None, **kwargs):
         fig.savefig(save_path, facecolor=fig.get_facecolor(), edgecolor="none")
 
     return fig, ax
+
+
+def plot_gammac(eq, rhos=None, alphas=None, num_pitch=None):
+    """Plot the energetic proxy metric γ_c fo a single flux surface.
+
+    Parameters
+    ----------
+    eq : object
+        Equilibrium object containing magnetic field information
+    rhos : array_like or float, optional
+        Flux surface radii. If float, plots single surface.
+        Default: np.linspace(0.1, 1, 10)
```

Comment:
below it looks like the default is rho=0.5

Reply:
also the code below looks like it only works for a single flux surface, not multiple. This is fine, just make it clear in the docstring and add a check inside that rho is a scalar.

Should also make it possible for the user to pass in an axis to plot to, so they can manually plot multiple surfaces as subplots